safety-quiz/templates/login.html

```diff
@@ -1,26 +1,58 @@
 {% extends 'layout.html' %}
 {% block page %}
+<script src="//cdnjs.cloudflare.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
+
 <div class="card card-default">
     <div class="card-header">
         Login
     </div>
-    <div class="card-body" {% if legacy == True %} hidden {% endif %}>
-        <a class="btn btn-lg btn-success" href="{{ url_for( 'auth.login_google') }}">Click here to login with Google.</a>
+    <div class="card-body" {% if legacy== True %} hidden {% endif %}>
+        <a class="btn btn-lg btn-success" href="{{ url_for( 'auth.login_google') }}">Click here to login with
+            Google.</a>
         <a href="?legacy=true">Legacy login</a>
     </div>
-    <div class="card-body" {% if legacy == False %} hidden {%  endif %}>
+    <div class="card-body" {% if legacy== False %} hidden {% endif %}>
```

Comment:
try to reset areas where only whitespace is changed. This (and the above change) didn't change the functionality of the code.